.github/workflows/run-e2e.yml

```diff
@@ -95,16 +92,12 @@ jobs:
       - run: yarn tsx .github/scripts/git-diff-default-branch.ts
         if: ${{ steps.download-changed-files.outcome == 'failure' }}
 
-      - name: Get job id
-        id: get-job-id
-        run: yarn tsx .github/scripts/get-job-id.ts
-
       - name: Run E2E tests
         timeout-minutes: ${{ inputs.test-timeout-minutes }}
         run: ${{ inputs.test-command }} --retries 1
         env:
           # The properties are picked up by mocha-junit-reporter and added to the XML test results
-          PROPERTIES: 'JOB_NAME:${{ env.JOB_NAME }},JOB_ID:${{ steps.get-job-id.outputs.job-id }},RUN_ID:${{ env.RUN_ID }}'
+          PROPERTIES: 'JOB_NAME:${{ env.JOB_NAME }},RUN_ID:${{ env.RUN_ID }},PR_NUMBER:${{ env.PR_NUMBER }}'
```

Comment:
Do you have to store PR_NUMBER here?  Won't it be the same when you pick up the XML files while generating the test report?  Or is this somehow for the flaky test report?